src/include/binder/bind/bound_drop_column.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "binder/bound_statement.h"
+
+namespace kuzu {
+namespace binder {
+
+class BoundDropColumn : public BoundStatement {
```

Comment:
Consider adding a BoundDDL base class for all DDL statements.

A separate comments that is not related to this PR. I notice the folder name seems to be wrong. It should "ddl" rather than "bind". Can u mimic the structure as in the parser module to rename this folder to "ddl" and "copy_csv"?

Reply:
Refactored the structure of the bound ddl folder. BoundDDL statements don't have any common fields. 
Added a BoundDDL base class